packages/node_modules/@webex/webex-core/src/lib/services/service-catalog.js

```diff
@@ -180,9 +159,11 @@ const ServiceCatalog = AmpState.extend({
    * @param {string} params.clusterId - clusterId of found service
    * @param {boolean} [params.priorityHost] - returns priority host url if true
    * @param {string} [params.serviceGroup] - specify service group
-   * @returns {object} - {name: string, url: string}
+   * @returns {object} service
+   * @returns {string} service.name
+   * @returns {string} service.url
    */
-  findServiceFromClusterId({clusterId, priorityHost, serviceGroup}) {
+  findServiceFromClusterId({clusterId, priorityHost = true, serviceGroup}) {
```

Comment:
It's nice to have `... = {}` at the end of the parameters just in case someone calls `findServiceFromCluserId()` with nothing.

Reply:
Implimented.